pincer/middleware/interaction_create.py

```diff
@@ -41,23 +42,41 @@ def get_command_from_registry(interaction: Interaction):
         The command is not registered
     """
 
+    name = interaction.data.name
```

Comment:
clearly missing typing here

Reply:
Does mypy really make you do that :skull: